AssetEditor/ViewModels/MenuBarViewModel.cs

```diff
@@ -277,10 +277,12 @@ void OpenAssetEditorFolder()
         void CompileAudioProjects() => _uiCommandFactory.Create<OpenEditorCommand>().Execute<CompilerViewModel>();
         void OpenTechSkeletonEditor() => _uiCommandFactory.Create<OpenEditorCommand>().Execute<EditorHost<SkeletonEditorViewModel>>();
         void OpenAnimationBatchExporter() => _uiCommandFactory.Create<OpenAnimationBatchConverterCommand>().Execute();
+        
         private void RunDialogueEventTesting()
         {
-            //var DialogueEventGenerator = new DialogueEventGenerator(_packfileService);
-            //DialogueEventGenerator.DoDialogueEventGenerator();
+            var mainWindow = new Editors.AudioEditor.MainWindow();
```

Comment:
Avoid using UI related code directly (MainWindow) as it makes automatic testing (when I get around to it...) nearly impossible